src/server/modes/charts/plugins/ql/preparers/metric.ts

```diff
@@ -98,32 +108,61 @@ export default ({
             value = parseNumberValue(value);
         }
 
-        result = [
-            {
-                content: {
-                    current: {
-                        value,
-                    },
+        const app = registry.getApp();
+
+        const useMarkupMetric = isEnabledServerFeature(app.nodekit.ctx, Feature.MarkupMetric);
+
+        if (useMarkupMetric) {
+            const size = 'm';
+            const color = 'rgb(77, 162, 241)';
+            const title = measureColumn.name;
+
+            let formattedValue = String(value);
+
+            if (typeof value === 'number') {
+                formattedValue = formatNumber(value, {});
+            }
+
+            return {
+                value: {
+                    type: 'concat',
+                    className: `markup-metric markup-metric_size_${size}`,
+                    children: [
+                        {
+                            className: 'markup-metric-title',
+                            type: 'text',
+                            content: title,
+                        },
+                        {
+                            type: 'color',
+                            color,
+                            content: {
+                                className: 'markup-metric-value',
+                                type: 'text',
+                                content: formattedValue,
+                            },
+                        },
+                    ],
```

Comment:
It looks almost identical to https://github.com/datalens-tech/datalens-ui/blob/31782815a7be290a5760be2a765b61768bf0bfab/src/server/modes/charts/plugins/datalens/preparers/metric/variants/markup.ts#L88
It would be healthy to share the code somehow